rules/windows/privilege_escalation_installertakeover.toml

```
@@ -16,35 +16,52 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential Privilege Escalation via InstallerFileTakeOver"
-note = """## Triage and analysis.
+note = """## Triage and analysis
 
-### Investigating Potential Priivilege Escalation via InstallerFileTakeOver
+### Investigating Potential Privilege Escalation via InstallerFileTakeOver
 
-InstallerFileTakeOver is a weaponized EoP PoC to the CVE-2021-41379 vulnerability. Upon successful exploitation,
-an unprivileged user will escalate privileges to SYSTEM/NT AUTHORITY.
+InstallerFileTakeOver is a weaponized EoP PoC to the CVE-2021-41379 vulnerability. Upon successful exploitation, an
```

I guessed at this acronym -- please check to make sure it's correct.
```suggestion
InstallerFileTakeOver is a weaponized escalation of privilege proof of concept (EoP PoC) to the CVE-2021-41379 vulnerability. Upon successful exploitation, an
```